controls/patches.rb

```diff
@@ -18,12 +20,12 @@
 control 'patches' do
   impact 0.3
   title 'All operating system package updates are installed'
-  linux_update.updates.each { |update|
+  linux_update.updates.each do |update|
     describe package(update['name']) do
       its('version') { should eq update['version'] }
     end
-  }
-  only_if { linux_update.updates.length > 0 }
+  end
+  only_if { linux_update.updates.length.positive? }
```

Comment:
nit: I am not fan of this style, but I assume we want to harmonize it, so its okay.